client/styles/components/_sketch-list.scss

```diff
@@ -33,4 +33,29 @@
   @include themify() {
     @extend %icon;
   }
-}
\ No newline at end of file
+}
+
+.visibility-toggle .sketch-list__trash-button {
+  @include themify() {
+    margin: 0;
+    padding: 0;
+    visibility: hidden;
```

Comment:
`visibility: hidden` is not accessibility friendly. TBH it's a bit of a hack to make visibly hidden elements accessible, but look at the placeholder `%hidden-element` in `_placeholders.scss`.